tracing.go

```diff
@@ -22,32 +22,37 @@ package tchannel
 
 import (
 	"fmt"
-	"math/rand"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"golang.org/x/net/context"
```

Comment:
This repo uses (or at least is supposed to use): std libs, local imports, external imports. Don't know if it's documented anywhere.

CC @prashantv